src/SupportMessage/__tests__/SupportMessage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+
+import { SupportMessage } from '../SupportMessage'
+
+describe('SupportMessage component', () => {
+  it('default renders correctly', () => {
```

Comment:
I think all these tests are basically doing exactly the same thing so not sure how worth they are?
Probably better to start avoiding using snapshots for new components and test them properly with testing library?